salt/cilium/init.sls

```diff
@@ -0,0 +1,16 @@
+include:
```

Comment:
I would move this `include` inside the `if` (so nothing is done if `cilium` is not enabled) and [add a dummy target ](https://github.com/kubic-project/salt/blob/crio-cleaups/salt/etc-hosts/init.sls#L11-L16) for the `else`